aspnetcore/blazor/tooling.md

```diff
@@ -690,37 +690,68 @@ The .NET WebAssembly build tools are based on [Emscripten](https://emscripten.or
 
+To enable AOT, set `<RunAOTCompilation>` to `true` in the app's project file (`.csproj`):
+
 ```xml
 <PropertyGroup>
   <RunAOTCompilation>true</RunAOTCompilation>
 </PropertyGroup>
 ```
 
-:::moniker-end
+To disable SIMD, for example when targeting old browsers (on mobile devices), add the `<WasmEnableSIMD>` property set to false:
 
-:::moniker range=">= aspnetcore-7.0 < aspnetcore-8.0"
+```xml
+<PropertyGroup>
+  <RunAOTCompilation>true</RunAOTCompilation>
+  <WasmEnableSIMD>false</WasmEnableSIMD>
+</PropertyGroup>
+```
 
-When [ahead-of-time (AOT) compilation](xref:blazor/host-and-deploy/webassembly#ahead-of-time-aot-compilation) is used, [WebAssembly Single Instruction, Multiple Data (SIMD)](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md) is supported for all major browsers. SIMD can improve the throughput of vectorized computations by performing an operation on multiple pieces of data in parallel using a single instruction. Use the `<WasmEnableSIMD>` property in the app's project file (`.csproj`) with a value of `true`:
+<!-- UPDATE 8.0 Remove note when https://github.com/dotnet/runtime/issues/89302 is fixed -->
+
+> [!IMPORTANT]
+> During 8.0 preview releases, disabling SIMD isn't supported. For more information, see [SIMD cannot be disabled (dotnet/runtime #89302)](https://github.com/dotnet/runtime/issues/89302).
+
+For more information, see [Configuring and hosting .NET WebAssembly applications: SIMD - Single instruction, multiple data](https://github.com/dotnet/runtime/blob/main/src/mono/wasm/features.md#simd---single-instruction-multiple-data).
+
+Exception handling is also enabled by default. To disable exception handling, add the `<WasmEnableExceptionHandling>` property with a value of `false`:
 
 ```xml
 <PropertyGroup>
-  <WasmEnableSIMD>true</WasmEnableSIMD>
   <RunAOTCompilation>true</RunAOTCompilation>
+  <WasmEnableExceptionHandling>false</WasmEnableExceptionHandling>
 </PropertyGroup>
 ```
 
 :::moniker-end
 
-:::moniker range=">= aspnetcore-7.0"
+:::moniker range=">= aspnetcore-7.0 < aspnetcore-8.0"
 
-To enable WebAssembly exception handling, use the `<WasmEnableExceptionHandling>` property in the app's project file (`.csproj`) with a value of `true`:
+When [ahead-of-time (AOT) compilation](xref:blazor/host-and-deploy/webassembly#ahead-of-time-aot-compilation) is used, [WebAssembly Single Instruction, Multiple Data (SIMD)](https://github.com/WebAssembly/simd/blob/master/proposals/simd/SIMD.md) is supported for all major browsers. SIMD can improve the throughput of vectorized computations by performing an operation on multiple pieces of data in parallel using a single instruction.
```

Comment:
AOT is orthogonal concept to SIMD, even Mono interpreter (no-AOT build) could use SIMD & EH instructions.

Reply:
I see ... sounds like I should split out SIMD into its own section.

What about EH? ... What is that related to (i.e., which section would that appear in)?

Reply:
both EH and SIMD are instructions which make things faster.
both could be used with and without AOT.
They are both enabled in Net8.

So, I think that SIMD and EH sections are on the same level as AOT, not subsection.